### DISCUSSION TITLE
What is that word?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#520] [Galaxy Trivia] [answered]
Each of these words can be split in half, and two letters can be placed in the middle to create a new six-letter word.

For example, BEND can be split into BE and ND, YO can be placed in the middle to create BE + YO + ND = BEYOND.

The four added pairs of letters, in order, spell an 8-letter word

LEND
LIAR
THAT
COIN

**Comment by peterbaccaro** [comment, marked as answer]
LE**GE**ND
LI**NE**AR
TH**RO**AT
CO**US**IN

**GENEROUS**

**Comment by ErKiran** [comment]
The four added pairs of letters, in order, spell the 8-letter word "ELECTRON".

Here's how the words can be split and combined to form the new word:

LEND -> LE + ND -> LEND
LIAR -> LI + AR -> L I AR
THAT -> TH + AT -> T H AT
COIN -> CO + IN -> C O IN
Combining the pairs of letters in order, we get "LETHCION". Rearranging the letters, we get "ELECTRON", which is an 8-letter word.
